test/e2e/workflow/deployment_test.go

```diff
@@ -198,3 +231,51 @@ func ignoreInitialKubeMacPoolRestart() {
 	By("Ignoring initial KubeMacPool restart")
 	time.Sleep(10 * time.Second)
 }
+
+func retreiveRange(configMap *v1.ConfigMap) (string, string) {
```

Comment:
Use gomega for error handling, you can use offset to show test line.

Reply:
do you mean I should use "EventuallyWithOffset" , because inside the Eventually function itself I don't think I can use "Expect" and such